Tests/Microsoft.Azure.Mobile.Test.UWP/Ingestion/Http/IngestionHttpTest.cs

```diff
@@ -0,0 +1,121 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Net;
+using System.Net.Http;
+using System.Threading;
+using System.Threading.Tasks;
+using HyperMock;
+using Microsoft.Azure.Mobile.Ingestion;
+using Microsoft.Azure.Mobile.Ingestion.Http;
+using Microsoft.Azure.Mobile.Ingestion.Models;
+using Microsoft.Rest;
+using Microsoft.VisualStudio.TestPlatform.UnitTestFramework;
+
+namespace Microsoft.Azure.Mobile.Test.Ingestion.Http
+{
+	[TestClass]
+	public class IngestionHttpTest
+	{
+		private Mock<IHttpNetworkAdapter> _adapter;
+		private IngestionHttp _ingestionHttp;
+
+		[TestInitialize]
+		public void InitializeIngestionHttpTest()
+		{
+			_adapter = Mock.Create<IHttpNetworkAdapter>();
+			_ingestionHttp = new IngestionHttp(_adapter.Object);
+		}
+		
+		[TestMethod]
```

Comment:
Could you please add ///\<summary\> style comments to the test methods to briefly describe what they are trying to test?